steward/src/commands/schedule_corks/aave2_clr_s/fees_distributor.rs

```diff
@@ -0,0 +1,61 @@
+use crate::{application::APP, cellars, cork, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use deep_space::Address;
+use ethers::abi::AbiEncode;
+use steward_abi::aave_v2_stablecoin::*;
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Set fees distributor of target Cellar.\n This command will set the fees distributor of a Cellar to a Cosmos address that the contract will bridge funds to. This is a validator only command and can only be run by validators. It sets the fees distributor of a Cellar based on the height specified by \n the validators. Therefore, it'll execute the function when the chain reaches that height. This command also takes the fee distributor's address"
```

Comment:
Nit: there's some redundancy in here we don't need in these command descriptions. The scheduled corks are implicitly validator-only, and we really only need to mention arguments if there's something to discuss about them, the help messages will show them and their comments. Could be simplified to something like: "Set fees distributor Cosmos address of target cellar when chain reaches specified height." Similar note for the rest of the commands.